src/Confluent.Kafka/Metadata.cs

```diff
@@ -42,15 +57,17 @@ public Metadata(List<BrokerMetadata> brokers, List<TopicMetadata> topics, int or
         public List<BrokerMetadata> Brokers { get; }
 
         /// <summary>
-        ///     Information about every topic managed by in the cluster.
+        ///     Information about each topic in the cluster.
```

Comment:
Since this might be all topics, single requested topic, or locally used topics, this should probably say:
Information about requested topics in cluster